tests/conftest.py

```diff
@@ -26,6 +26,6 @@ def is_unittest(obj):
 
 def pytest_pycollect_makeitem(collector, name, obj):
     if is_unittest(obj) and not obj.__name__.startswith("_"):
-        return UnitTestCase(name, parent=collector)
+        return UnitTestCase.from_parent(name=name, parent=collector)
```

Comment:
we may want to pin pytest to minimum version 5.4.0 in tox.ini here as I believe that's when this was introduced.

Reply:
Done